randomizer.py

```diff
@@ -27,8 +27,8 @@ def __init__(self, options, *, seed=None):
         self.rnd = random.Random(self.seed)
         if options.race:
             self.rnd.random()  # Just pull 1 random number so race seeds are different then from normal seeds but still stable.
-        if options.goal == "random":
-            options.goal = self.rnd.randint(-1, 8)
+        if type(options.goal) is range:

```

Comment:
Better use `isinstance()` instead of `type()`

Reply:
fixing